lua/modules/item_editor.lua

```diff
@@ -1125,9 +1125,9 @@ initClientFunctions = function()
 		end
 
 		-- if facemode is enable, test the neighbor blocks of impact.Block
-		if addedBlock ~= nil and facemode == true then
```

Comment:
could we keep the `== true` please, otherwise it also succeeds if `facemode` is `1` or any table I think

Reply:
It can't be anything but a boolean https://github.com/cubzh/cubzh/pull/518/files#diff-cf6ede0b9349de179de77c358cf61f9febdda6b751456953b070078602de372eL1114-L1116